src/App.tsx

```diff
@@ -1,26 +1,176 @@
-/* eslint-disable max-len */
+/* eslint-disable @typescript-eslint/no-shadow */
+/* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
-
-const USER_ID = 0;
+import React, {
+  useCallback,
+  useEffect,
+  useMemo,
+  useRef,
+  useState,
+} from 'react';
+import { addTodo, deleteTodo, getTodos, USER_ID } from './api/todos';
+import { TodoForm } from './components/TodoForm/TodoForm';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Error } from './components/Error/Error';
+import { Todo } from './types/Todo';
+import { FilterOption } from './types/Filter';
+import { ErrorType } from './types/Error';
+import { Loader } from './components/Loader/Loader';
+import { filterTodos } from './utils/filter';
+import { TempTodoItem } from './components/TempTodoItem/TempTodoItem';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
-  }
+  // #region states
+
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [loading, setLoading] = useState(false);
```

Comment:
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```
boolean should start with` is has are `